OrbitVulkanLayer/DispatchTable.cpp

```diff
@@ -0,0 +1,205 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OrbitVulkanLayer/DispatchTable.h"
+
+#include "OrbitBase/Logging.h"
+
+namespace orbit::layer {
+
+void DispatchTable::CreateInstanceDispatchTable(const VkInstance& instance,
+                                                const PFN_vkGetInstanceProcAddr& next_gipa) {
+  // Create dispatch table
+  VkLayerInstanceDispatchTable dispatch_table = {};
```

Comment:
Does this `= {}` notation have a different effect than just `VkLayerInstanceDispatchTable dispatch_table`?